tutorecommerce/plugin.py

```diff
@@ -45,6 +45,7 @@
                 "merchant_id": "SET-ME-PLEASE",
                 "soap_api_url": "https://ics2wstest.ic3.com/commerce/1.x/transactionProcessor/CyberSourceTransaction_1.140.wsdl",
                 "transaction_key": "SET-ME-PLEASE",
+                "flex-run-environment": "cybersource.environment.sandbox",
```

Comment:
The variable name should be `flex_run_environment` like all the above variables instead of `flex-run-environment`.